README.md

```diff
@@ -1,8 +1,29 @@
-## Generate something from GIR
+# GIR
+
+The `GIR` is used to generate both sys level and Rust-user API level.
+
+## Generate FFI (sys level)
+
+You first need to get the Gir file corresponding to the project you want to bind. You can get them from [here](https://github.com/gtk-rs/gir-files) or directly on [ubuntu website](http://packages.ubuntu.com/) (for example: http://packages.ubuntu.com/zesty/amd64/libgtk-3-dev).
+
+When you're ready, let's generate the FFI part. Don't forget to download the gir files as well.
 
-Running:
 ```shell
-cargo run --release -- -c Gir_Gtk.toml -d ../gir-files -o gtk
+cargo run --release -c YourGirFile.toml -d ../gir-files -m sys -o the-output-directory-sys
 ```
-where `gir-files` contains the [GIR definitions](https://github.com/gkoz/gir-files).
-The generated files will be placed in `gtk`
+
+The generated files will be placed in `the-output-directory-sys`. You now have the sys part of your binding!
+
+## Generate the Rust-user API level
+
+Normally, a toml file corresponding to your crate has been generated (you can take a look to [gir-gtk.toml](https://github.com/gtk-rs/sys/blob/master/conf/gir-gtk.toml) as an example).
```

Comment:
gir-gtk.toml link belong to ffi part.
here better local `Gir_Gtk.toml` or gtk's Gir.toml